### PR TITLE
[INFINITY-944] Developer doc improvements

### PR DESCRIPTION
**Pull request by mgummelt**
cc @sascala

### CODE REVIEW

File: docs/pages/dev-guide/developer-guide.md [hunk 1]
```diff
@@ -38,6 +38,69 @@ Although all SDK services written today store metadata in Zookeeper, this is an
...
+
+1. View `svc.yml`
+
+   Take a look at `src/main/dist/svc.yml`.  This is the YAML file that
+   defines your framework.  You will be editing this file.
+
+1. View `Main.java`
+
+   Take a look at
+   `src/main/java/com/mesosphere/sdk/mgummelt/scheduler/Main.java`.
```
Comment (sascala) [inline docs/pages/dev-guide/developer-guide.md]: In the future, we should probably move this file out of a directory named after you.

File: docs/pages/dev-guide/developer-guide.md [hunk 2]
```diff
@@ -38,6 +38,69 @@ Although all SDK services written today store metadata in Zookeeper, this is an
...
+   This is the main method for your scheduler, which will be run in
+   DC/OS via Marathon.  It reads `svc.yml`, which defines its
+   behavior.  If you need any advanced functionality not provided by
+   YAML, such as complex deployment plans, you will write it here.
+
+1. Build a [package](#packaging)
```
Comment (sascala) [inline docs/pages/dev-guide/developer-guide.md]: add a period at the end of "(#packaging)".

File: docs/pages/dev-guide/developer-guide.md [hunk 3]
```diff
@@ -38,6 +38,69 @@ Although all SDK services written today store metadata in Zookeeper, this is an
...
+   ```bash
+   $ ./build.sh aws
+   ```
+
+   You will deploy your framework to DC/OS as a
+   [package](#packaging).  `build.sh` creates this package.
+
+1. Install your package.
+
+   `build.sh` should print instructions for installing the package.
```
Comment (sascala) [inline docs/pages/dev-guide/developer-guide.md]: suggest: "`builds` prints installation instructions."

File: docs/pages/dev-guide/developer-guide.md [hunk 4]
```diff
@@ -38,6 +38,69 @@ Although all SDK services written today store metadata in Zookeeper, this is an
...
+   They will look something like this:
+
+   ```bash
+   $ dcos package repo remove myframework-aws
+   $ dcos package repo add --index=0 myframework-aws https://mybucket.s3.amazonaws.com/stub-universe-myframework.zip
+   $ dcos package install --yes myframework
+   ```
+
+   You may now navigate to the DC/OS Services UI to view the
```
Comment (sascala) [inline docs/pages/dev-guide/developer-guide.md]: Remove "You may now"

File: docs/pages/dev-guide/developer-guide.md [hunk 5]
```diff
@@ -38,6 +38,69 @@ Although all SDK services written today store metadata in Zookeeper, this is an
...
+   deployment.
+
+1. Uninstall your package.
+
+   ```
+   $ dcos package uninstall myframework
+   $ dcos node ssh --master-proxy --leader "docker run mesosphere/janitor /janitor.py -r myframework-role -p myframework-principal -z dcos-service-myframework
+   ```
+
+   The second command listed above is known as the **janitor**.  It is
```
Comment (sascala) [inline docs/pages/dev-guide/developer-guide.md]: Suggest "The second command above runs the **janitor** script. The **janitor** script runs inside the DC/OS cluster to clean up Zookeeper state..."

Comma after "but, for now"

File: docs/pages/dev-guide/developer-guide.md (hunk 1 above)
Comment (mgummelt) [inline docs/pages/dev-guide/developer-guide.md]: lol, fixed

**Comment by mgummelt** [comment]
@sascala all comments addressed

**Pull request merged**